src/Compilers/VisualBasic/Portable/Compilation/VisualBasicCompilation.vb

```diff
@@ -3140,6 +3140,17 @@ Namespace Microsoft.CodeAnalysis.VisualBasic
             End Select
         End Function
 
+        Private Protected Overrides Function SupportsRuntimeCapabilityCore(capability As RuntimeCapability) As Boolean
+            Dim assembly = Me.Assembly
+
+            Select Case capability
+                Case RuntimeCapability.DefaultImplementationsOfInterfaces
+                    Return assembly.RuntimeSupportsDefaultInterfaceImplementation
+            End Select
+
+            Return False
```

Comment:
It doesn't seem obvious that we should return false here. 
The alternative is to accurately report what the runtime/TFM supports, regardless of whether the language cares.
I'm curious what others think.

Reply:
Good question.  @AlekseyTs what's your thinking here?

Reply:
> The alternative is to accurately report what the runtime/TFM supports, regardless of whether the language cares.

I think I am leaning towards this approach as well. It feels that whether language supports something and whether runtime supports something are orthogonal things. For example, runtime might support some feature, but it might be unavailable due to target language version. Another example is the fact that DIM feature is supported at different levels VB vs. C#. So, the IDE will likely need to take several different aspects into account (runtime support level, language version, language VB vs. C#, etc.) while deciding if it is appropriate to make certain suggestions. Also, it might be confusing to users when for the same runtime VB gives a different answer by comparison to C#.

Reply:
Alternatively we can rename the API to reflect the dependency (it is not purely about runtime) and clearly document it.

Reply:
> I think I am leaning towards this approach as well.

Sounds good.  Let me try moving to that approach.  Will ping when ready.